### DISCUSSION TITLE
Extension to fetch keys on specific pattern

### DISCUSSION THREAD

**Discussion by pwalecha** [microsoft/UnifiedRedisPlatform.Core#2] [Ideas]
We are looking out for a functionality to enable pattern specific search over keys. E.g. if the Key is "ABCDY", if I pass "ABC*" (wild character), it should do a startswith (starting with ABC*) search. Similarly, *ABC should enable searching the keys ending with ABC. We could also look at making use of Regex to enable the pattern search.

**Comment by patrickCode** [comment]
This feature is not available in both the core and DistributedCache library. Download the latest libraries from these locations:
- Core --> https://www.nuget.org/packages/UnifiedRedisPlatform/
- Distributed Cache --> https://www.nuget.org/packages/DistributedCache.Extensions.UnifiedRedisPlatform/

Samples
- Core --> https://github.com/microsoft/UnifiedRedisPlatform.Core/blob/main/src/tests/Microsoft.UnifiedRedisPlatform.TestApps/TestConsoleCore.Latest/Program.cs
- Distributed Cache --> https://github.com/microsoft/UnifiedRedisPlatform.Core/blob/main/src/tests/Microsoft.UnifiedRedisPlatform.TestApps/TestWebAppCore.Latest/Controllers/CacheController.cs
